packages/lisk-transactions/test/4_register_multisignature_account.ts

```diff
@@ -34,11 +36,11 @@ describe('#registerMultisignatureAccount transaction', () => {
 	const lifetime = 5;
 	const minimum = 2;
 
-	let tooShortPublicKeyKeysgroup;
-	let plusPrependedPublicKeyKeysgroup;
-	let keysgroup;
-	let getTimeWithOffsetStub;
-	let registerMultisignatureTransaction;
+	let tooShortPublicKeyKeysgroup: Array<string>;
+	let plusPrependedPublicKeyKeysgroup: Array<string>;
+	let keysgroup: Array<string>;
+	let getTimeWithOffsetStub: sinon.SinonStub;
+	let registerMultisignatureTransaction: BaseTransaction;
```

Comment:
I don't like that we have to use `BaseTransaction` type here. I think the transaction module should, at a minimum, provide specific types for exposed functions. What are your thoughts?